index.bs

```diff
@@ -369,6 +374,7 @@ All {{FontIterator}} objects contain an internal <dfn attribute for=FontIterator
 The <dfn iterator for=FontIterator>asynchronous iterator initialization steps</dfn> for {{FontIterator}} are as follows:
 
 1. Set [=/this=]'s {{FontIterator/[[FontList]]}} to a new empty [=/queue=].
+1. Obtain a sorted list of each local font |font|.
```

Comment:
Sorted how?

Reply:
We can use this infra definition (and see examples). https://infra.spec.whatwg.org/#list-sort-in-ascending-order